src/View/ViewableData.php

```diff
@@ -412,7 +412,7 @@ public function renderWith($template, $customFields = null)
     protected function objCacheName($fieldName, $arguments)
     {
         return $arguments
-            ? $fieldName . ":" . implode(',', $arguments)
+            ? $fieldName . ":" . var_export($arguments, true)
```

Comment:
Required as we can now pass actual booleans as arguments. Implode will cast `false` to an empty string, which would mean the same cache name between `$Foo` and `$Foo(false)`